src/librustdoc/html/static/css/rustdoc.css

```
@@ -1585,6 +1585,14 @@ details.rustdoc-toggle > summary:hover::before {
 	opacity: 1;
 }
 
+details.rustdoc-toggle > summary:focus-visible::before {
+	/* The SVG is black, and gets turned white using a filter.
+	   Do the same with the outline.
+	*/
```

This comment was confusing to me. I think it would be helpful to add context, and also talk about why the dotted outline:

```suggestion
	/* The SVG is black, and gets turned white using a filter in the dark themes.
	   Do the same with the outline.
	   The dotted 1px style is copied from Firefox's focus ring style.
	*/
```